opflexagent/config.py

```diff
@@ -30,6 +30,16 @@
                 default=['*'],
                 help=_("List of the physical networks managed by this agent. "
                        "Use * for binding any opflex network to this agent")),
+    cfg.ListOpt('firewall_map',
+                default=[('dvs_port_key',
```

Comment:
This is meant to be the port  key -- this value is the key used to select the DVS firewall driver, and not the port group (port groups will be dynamic, whereas the firewall drivers will be configured).